techcity/api/tests/test_views.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import reverse
+
+
+class TestGetGroups:
+    def test_ok(self, client):
```

Comment:
If you added a `GroupFactory` call to this test as part of the setup, that would exercise a lot more of the code and ensure that some content is actually being serialized. As is, this `response` would only have `b'[]'` as its content.

Reply:
Make sense to me, I added a few more assertions too.